.github/dependabot.yml

```diff
@@ -1,7 +0,0 @@
-version: 2
```

Comment:
Is this gone because the `ci.yml` is now generated right?

Reply:
Yes 👍